src/main/kotlin/endpoint/eth/EthRefund.kt

```diff
@@ -0,0 +1,21 @@
+package endpoint.eth
+
+/**
+ * Type of address in Ethereum
+ */
+typealias Address = String
+
+/**
+ * Type of coin in Ethereum. Possibly it will be an address
+ */
+typealias CoinType = String
+
+/**
+ * Type of amount in Ethereum
+ */
+typealias AmountType = Double
```

Comment:
I think amount in ether is BigInteger. It seems to me that it is better to move all Ethereum data types into separate file.

Reply:
Yeah, we have to investigate the type on eth side and put it here.